DepositeCalcTests/Tests/SettingsPageTests.cs

```diff
@@ -0,0 +1,165 @@
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var settingsPage = new SettingsPage(driver);
+
+            // Act
+            settingsPage.DateFormatDropDown = expectedDateFormat;
+            settingsPage.ClickOnSaveBnt();
+            driver.SwitchTo().Alert().Accept();
+            calculatorPage.StartDateYear = year;
+            calculatorPage.StartDateMonth = month;
+            calculatorPage.StartDateDay = day;
+            string expectedSeparator = expectedDateFormat.Substring(2, 1);
+            string expectedEndDayStartsWithDays = day + expectedSeparator + expectedMonthNumber + expectedSeparator + year;
+
+            // Assert
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(expectedSeparator, calculatorPage.EndDate.Substring(2, 1), "Incorrect separator in the end date field");
+                Assert.AreEqual(expectedEndDayStartsWithDays, calculatorPage.EndDate, "Incorrect date format in the end date field");
+            });
+        }
+
+        [TestCase("MM/dd/yyyy", "2024", "April", "10", "04")]
+        [TestCase("MM dd yyyy", "2024", "September", "25", "09")]
+        public void DateFormatStrartWithMonthTest(string expectedDateFormat, string year, string month, string day, string expectedMonthNumber)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var settingsPage = new SettingsPage(driver);
+
+            // Act
+            settingsPage.DateFormatDropDown = expectedDateFormat;
+            settingsPage.ClickOnSaveBnt();
+            driver.SwitchTo().Alert().Accept();
+            calculatorPage.StartDateYear = year;
+            calculatorPage.StartDateMonth = month;
+            calculatorPage.StartDateDay = day;
+            string expectedSeparator = expectedDateFormat.Substring(2, 1);
+            string expectedEndDayStartsWitMonth = expectedMonthNumber + expectedSeparator + day + expectedSeparator + year;
+
+            // Assert
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(expectedSeparator, calculatorPage.EndDate.Substring(2, 1), "Incorrect separator in the end date field");
+                Assert.AreEqual(expectedEndDayStartsWitMonth, calculatorPage.EndDate, "Incorrect date format in the end date field");
+            });
+        }
+
+        [TestCase("$ - US dollar")]
+        [TestCase("€ - euro")]
+        [TestCase("£ - Great Britain Pound")]
+        [TestCase("₴ - Ukrainian hryvnia")]
+        public void DefaultCurrencyTest(string expectedCurrency)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+            var settingsPage = new SettingsPage(driver);
+
+            // Act
+            settingsPage.DefaultCurrencyDropDown = expectedCurrency;
+            settingsPage.ClickOnSaveBnt();
+            driver.SwitchTo().Alert().Accept();
```

Comment:
What do you think about a method on SettingsPage:

```
public void Set(string currency = null, string dateFormat = null, string numberFormat = null)
{
    if (currency != null) DefaultCurrencyDropDown = currency;
    if (dateFormat != null) DateFormatDropDown = dateFormat;
    if (numberFormat != null) NumberFormatDropDown = numberFormat;
    Save();
}
```

So in test it will be:
```suggestion
            settingsPage.Set(expectedCurrency);
```

and for `numberFormat`:
```
            settingsPage.Set(numberFormat: expectedNumberFormat);
```